packages/@okta/i18n/src/properties/login_fi.properties

```diff
@@ -930,6 +940,14 @@ oie.symantecVip.challenge.title = Todenna seuraavalla tavalla: {0}
 oie.symantecVip.challenge.description = Anna {0} -sovelluksessa luotu turvakoodi.
 oie.symantecVip.verify.passcode.label = Sy\u00F6t\u00E4 turvakoodi
 
+## YubiKey Authenticator
+oie.yubikey.label = YubiKey Authenticator
```

Comment:
Interesting seeing this translated in only a few languages. Let's confirm that this is correct before merging.

Reply:
The English loan word "authenticator" entered the technical term lexicon in these languages. This is expected.